README.md

```diff
@@ -1,2 +1,7 @@
 # gk-installer
-Installer for gluster/gluster-kubernetes
+Installer for [gluster/gluster-kubernetes](https://github.com/gluster/gluster-kubernetes)
+
+## Usage
+The installer expects that you are running the container in the Kubernetes cluster you intend to deploy gluster-kubernetes to. It also expects that you will mount your topology file to /etc/gk-deploy/topology.json (i.e. via ConfigMap)
+
+The repo includes an example ConfigMap and Job for deployment
```

Comment:
Please add a link to https://github.com/heketi/heketi/wiki/Setting-up-the-topology